odtp/dashboard/utils/storage.py

```diff
@@ -226,3 +229,35 @@ def storage_run_selection(execution_id, repo_url, commit_hash):
         app.storage.user["run_selection"] = json.dumps(run_selection)
     except Exception as e:
         ui.notify(f"storage update for run selection failed: {e}", type="negative")
+    
+def app_storage_is_set(value):
+    storage_entry_for_value = app.storage.user.get(value)
+    if storage_entry_for_value == "None" or not storage_entry_for_value:
+        return False
+    return True
+
+
+def storage_update_user(user_data):
+    try:
```

Comment:
fix: I think it is important to store the user after he authenticated. Therefore you should not do this in a try except block, but rather raise error in case anything goes wrong here.